packages/codemod-utils/src/angularHelpers.ts

```diff
@@ -77,6 +78,14 @@ type Property = Types.Property;
 type TemplateLiteral = Types.TemplateLiteral;
 
 const ANGULAR_PACKAGE_NAME = '@angular/core';
+
+const ANGULAR_PACKAGE_NAME_MATCHER: ImportedModuleMatcher = {
+  importModulePattern: ANGULAR_PACKAGE_NAME,
+  importUmdPattern: null,
+  framework: 'vanilla',
```

Comment:
Is this meant to be vanilla?

Reply:
ohhh you are right, it doesn't matter here but this should be "angular". fixing.